src/uas/UAS.h

```diff
@@ -357,7 +357,6 @@ public slots:
     bool componentMulti[256];
     bool connectionLost; ///< Flag indicates a timed out connection
     quint64 connectionLossTime; ///< Time the connection was interrupted
-    quint64 lastVoltageWarning; ///< Time at which the last voltage warning occurred

```

Comment:
the uas class is deprecated. i put this lastvoltagewarning in the vehicle class.
originally our widgets were based on the asluav class which inherits from uas. since uas is deprecated i decided to just put everything into vehicle